mkt/webapps/serializers.py

```diff
@@ -212,6 +213,13 @@ def get_ratings_aggregates(self, app):
         return {'average': app.average_rating,
                 'count': app.total_reviews}
 
+    def get_size(self, app):
+        """App size whether it be size of a package or size of a hosted app."""
+        try:
+            return app.current_version.files.latest().size
```

Comment:
if you use `app.current_version.all_files[0].size`, the `all_files` method is heavily cached.